tbx/core/templatetags/util_tags.py

```diff
@@ -88,3 +91,21 @@ def ifinlist(value, list):
     # cast to strings before testing as this is used for heading levels 2, 3, 4 etc
     stringList = [str(x) for x in list]
     return str(value) in stringList
+
+
+@register.filter(name="has_raw_html_block")
+def has_raw_html_block(value):
+    if not isinstance(value, StreamValue):
+        return False
+
+    for block in value._raw_data:
+        if block["type"] == "raw_html" and "https://gist.github.com" in block["value"]:
+            return True
+    return False
+
+
+@register.filter(name="has_markdown_block")
+def has_markdown_block(value):
+    if isinstance(value, StreamValue):
+        return any(type(item.block) is MarkdownBlock for item in value)
```

Comment:
```suggestion
    if not isinstance(value, StreamValue):
        return False
       
    for block in value._raw_data:
        if block["type"] == "markdown":
            return True
```

for another early exit.
With this, you no longer need the

```
from wagtail import blocks
from wagtailmarkdown.blocks import MarkdownBlock
```